downstream/titles/release-notes/async/aap-25-20250507.adoc

```diff
@@ -0,0 +1,310 @@
+[[aap-25-20250507]]
+
+= {PlatformNameShort} patch release May 7, 2025
+
+This release includes the following components and versions:
+
+[cols="1a,3a", options="header"]
+|===
+| Release date | Component versions
+
+| May 7, 2025 | 
+* {ControllerNameStart} 4.6.12
+* {HubNameStart} 4.10.4
+* {EDAName} 1.1.8
+* Container-based installer {PlatformNameShort} (bundle) 2.5-13
+* Container-based installer {PlatformNameShort} (online) 2.5-13
+* Receptor 1.5.5
+* RPM-based installer {PlatformNameShort} (bundle) 2.5-12
+* RPM-based installer {PlatformNameShort} (online) 2.5-12
+
+|===
+
+CSV Versions in this release:
+
+* Namespace-scoped Bundle: aap-operator.v2.5.0-0.1746137767
+
+* Cluster-scoped Bundle: aap-operator.v2.5.0-0.1746138413
+
+
+== General
+
+* Implemented GitHub application credential type.(AAP-38589)
+
+* The `ansible.platform` collection has been updated to 2.5.20250507.(AAP-44992)
+
+* The `ansible.controller` collection has been updated to 4.6.12.
+
+* The `ansible.eda` collection has been updated to 2.7.0.
+
+== Features
+
+=== {PlatformNameShort}
+
+* Adds an enhanced log viewer for rulebook activation instances similar to the job output logger.(AAP-43337)
```

Comment:
This is very minor and you can wait to handle it later, if preferred, but I'd be consistent with the verbs here since you're using past tense: 
```suggestion
* Added an enhanced log viewer for rulebook activation instances similar to the job output logger.(AAP-43337)
```